Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -148,10 +148,10 @@ typedef void(^VOKManagedObjectsReturnBlock)(VOKArrayOfManagedObjects *managedObj
                                forManagedObjectContext:(nullable NSManagedObjectContext *)contextOrNil;
 
 /*
- Returns one entite matching the predicate. Asserts the count is exactly 1. If more objects are returned this method will let you know.
+ Returns one entity matching the predicate. Asserts the count is exactly 1. If more objects are returned, the last one returned by Core Data will be returned.
```

Comment:
That's part of the reason I removed it: "this method will let you know" isn't really descriptive, and it already explains the assertion. I just wanted to clarify what would happen in a release build.